### PR TITLE
Remove the old Mac debugger

### PR DESCRIPTION
**Pull request by TsvetanMilanov**
Remove Xillium and use Safari's webinspector for iOS and Chrome's inspector for Android.

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by justcodebuilduser** [comment]
:broken_heart:

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by justcodebuilduser** [comment]
:heart:

File: mobile/mobile-core/android-device-discovery.ts [hunk 1]
```diff
@@ -34,9 +34,12 @@ export class AndroidDeviceDiscovery extends DeviceDiscovery implements Mobile.IA
 	}
 
 	public startLookingForDevices(): IFuture<void> {
+		return (() => {
+
 		this.ensureAdbServerStarted().wait();
```
Comment (rosen-vladimirov) [inline mobile/mobile-core/android-device-discovery.ts]: tab to the right

File: definitions/mobile.d.ts [hunk 2]
```diff
@@ -267,6 +267,7 @@ declare module Mobile {
 		stopDeviceDetectionInterval(): void;
 		getDeviceByIdentifier(identifier: string): Mobile.IDevice;
 		mapAbstractToTcpPort(deviceIdentifier: string, appIdentifier: string): IFuture<string>;
+		detectCurrentlyAttachedDevices(): IFuture<void> ;
```
Comment (rosen-vladimirov) [inline definitions/mobile.d.ts]: please remove the space before `;`

File: mobile/mobile-core/devices-service.ts [hunk 3]
```diff
@@ -195,7 +195,7 @@ export class DevicesService implements Mobile.IDevicesService {
 		return searchedDevice;
 	}
 
-	private startLookingForDevices(): IFuture<void> {
+	public startLookingForDevices(): IFuture<void> {
```
Comment (rosen-vladimirov) [inline mobile/mobile-core/devices-service.ts]: this should be private

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by justcodebuilduser** [comment]
:heart:

**Pull request merged**